src/libstore/s3-binary-cache-store.md

```diff
@@ -2,7 +2,104 @@ R"(
 
 **Store URL format**: `s3://`*bucket-name*
 
-This store allows reading and writing a binary cache stored in an AWS
-S3 bucket.
+This store allows reading and writing a binary cache stored in an AWS S3 (or S3-compatible service) bucket.
+This store shares many idioms with the [HTTP Binary Cache Store](#http-binary-cache-store`).
+
+For AWS S3, the binary cache URL for a bucket named `example-nix-cache` will be exactly <s3://example-nix-cache>.
+For S3 compatible binary caches, consult that cache's documentation.
+
+### Anonymous Reads to your S3-compatible binary cache
+
+> If your binary cache is publicly accessible and does not require authentication,
+> the simplest and easiest way to use Nix with your S3 compatible binary cache is to use the HTTP URL for that cache.
+> In other words, the simplest way to configure Nix to use [HTTP Binary Cache Store] rather than S3 Binary Cache Store;
+> <https://example-nix-cache.s3.amazonaws.com> instead of <s3://example-nix-cache>.
```

Comment:
```suggestion
> it is simplest to use the [HTTP Binary Cache Store] rather than S3 Binary Cache Store with
> <https://example-nix-cache.s3.amazonaws.com> instead of <s3://example-nix-cache>.
```